### DISCUSSION TITLE
How to use the MauiCompat in new MAUI app?

### DISCUSSION THREAD

**Discussion by IeuanWalker** [xamarin/XamarinCommunityToolkit#1740] [Q&A] [answered]
Starting to look at MAUI and want to use some features in the community toolkit.
So thought i could just install `Xamarin.CommunityToolkit.MauiCompat` until all the features have been copied into the official MAUI nuget.

![image](https://user-images.githubusercontent.com/6544051/143612118-43c8316b-3ea6-45dd-b2ce-12136fe85a6f.png)

But I'm getting an error saying 'The type or namespace name 'CommunityToolkit' does not exist in the namespace 'Xamarin' (are you missing an assembly reference?)	FitNote (net6.0-ios), FitNote (net6.0-maccatalyst)' - 
![image](https://user-images.githubusercontent.com/6544051/143611845-fc19ea0d-fa3c-4a45-8350-7cf0f4c6136a.png)

It does look like it's working for android, but cant test as VS sees it as a build error - 
![image](https://user-images.githubusercontent.com/6544051/143611824-aaff6dd0-92d9-4148-b278-4f4c25a2158c.png)

Am I doing something wrong or is there a workaround for this? Or is it just not ready to be used?

**Comment by pictos** [comment, marked as answer]
Did we forget to add some `#if`?
cc: @brminnick @VladislavAntonyuk

**Reply by VladislavAntonyuk** [reply]
it should be available: https://github.com/xamarin/XamarinCommunityToolkit/blob/Xamarin.CommunityToolkit.MauiCompat-1.3.0/src/CommunityToolkit/Xamarin.CommunityToolkit.MauiCompat/ObjectModel/ObservableObject.shared.cs

**Reply by pictos** [reply]
@IeuanWalker can you open a bug issue and attach a small sample?

**Reply by IeuanWalker** [reply]
@pictos @VladislavAntonyuk issue with repo created - #1741
